### ISSUE TITLE
CPS color issue

### ISSUE THREAD

**Issue by fonkfader** [tiduscrying/MiniUI-Extra-Extras#2]
hi,
thanks a lot for this packs but i having issue with CPS1/2/3 (using fbalpha_libretro and fbalpha2012_cps3), the games launch ok but i have wrong color :

![progear_000](https://user-images.githubusercontent.com/1255088/166419807-8a24188d-8123-4af1-b6ac-607dfdffd249.png)

i don't know if this come from wrong romset (but i have tryed different one with same result) 

- miniUI : MiniUI-20220501-0
- MiniUI-Extra-Extras : latest zipped from source 

log : 

```
INFO: buffer_init()
 _MI_AO_OpenVqeLib: success
 _MI_AO_OpenSrcLib: success
 _MI_AO_OpenG711Lib: success
 _MI_AO_OpenG726Lib: success
 _MI_AO_OpenVqeLib: success
 _MI_AO_OpenSrcLib: success
 _MI_AO_OpenG711Lib: success
 _MI_AO_OpenG726Lib: success
input: new device #0 "sdl:keys"
input: async-only devices detected..
# drv probed binds name
0   0      y     y sdl:keys
create miao audio device...
created miao audio device!
open miao audio
Settings client
brightness: 10
speaker: 17
buffer size: 2048
INFO: Using nearest resampler (in: 0, out: 22050)
libpicofe/readpng.c: failed to open: /mnt/SDCARD/.system/bin/skin/font.png
libpicofe/readpng.c: failed to open: /mnt/SDCARD/.system/bin/skin/selector.png
INFO: Loading core /mnt/SDCARD/Emus/FBA.pak/fbalpha_libretro.so
INFO: Using nearest resampler (in: 0, out: 22050)
INFO: Finished loading core
INFO: Loading /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear.zip
INFO: Setting save dir to /mnt/SDCARD/Saves/FBA/
INFO: Setting system dir to /mnt/SDCARD/Bios/FBA
INFO: Using nearest resampler (in: 0, out: 22050)
INFO: [FBA] Archive: progear
INFO: [FBA] Using ROM with good CRC and name pgau.03 from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pgau.04 from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.01c from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.01d from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.01a from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.01b from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.03c from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.03d from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.03a from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.03b from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga.01 from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.05a from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.05b from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.06a from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name pga-simm.06b from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: [FBA] Using ROM with good CRC and name progear.key from archive /mnt/SDCARD/Roms/Final Burn Alpha (FBA)/progear
INFO: Using nearest resampler (in: 0, out: 22050)
INFO: Initialize DIP switches.
INFO: Using nearest resampler (in: 0, out: 22050)
INFO: [Cheevos] System RAM set to 0xb00af008 65536
INFO: Driver progear was successfully started
INFO: Screen: 384x224
INFO: Audio sample rate: 48002.148438
INFO: Frame rate: 59.629997
INFO: Reported aspect ratio: 1.333333
close miao audio
create miao audio device...
created miao audio device!
open miao audio
Settings client
brightness: 10
speaker: 17
buffer size: 2048
INFO: Using nearest resampler (in: 48002, out: 48000)
Settings client
brightness: 10
speaker: 17
GFX_init
ERROR: Error opening state file: No such file or directory
INFO: Dimensions changed to 384x224
INFO: Unloading core...
close miao audio
```

 thanks for your help

**Comment by tiduscrying** [comment]
Hi there! I believe the issue is either with FBA or with how the MiniUI native scaler is handled. I've been able to get around the issue by just setting the video mode to "Aspect" in the advanced menu. I know it's not the best solution, but it resolved the majority of color issues with the CPS1 and CPS2 ROMS that I've tried.

![screenshot_007](https://user-images.githubusercontent.com/17381025/166487977-4fef6397-46d5-4192-9a66-f274ae6e0b7e.png) 
![screenshot_005](https://user-images.githubusercontent.com/17381025/166487961-67c2b5d9-c9a4-46e4-bb30-23a8f5b548cc.png)
